packages/firebase/index.d.ts

```diff
@@ -2610,6 +2670,29 @@ declare namespace firebase.auth {
      * Signs out the current user.
      */
     signOut(): Promise<void>;
+    /**
+     * The current Auth instance's tenant ID. This is a readable/writable
+     * property. When set the enant ID on a Auth instance, all future
```

Comment:
When you set the tenant ID of an Auth instance, all future sign-in/sign-up operations will pass this tenant ID and sign in or sign up users to the specified tenant project.